addons/pkg/util/clusterbootstrapclone/clusterbootstrapclone.go

```diff
@@ -62,6 +63,10 @@ func NewHelper(ctx context.Context, k8sClient client.Client, aggregateAPIResourc
 // a fully qualified name(<packageShortName>.<domain>.<version>). The fully qualified name is fetched from TKR.
 func (h *Helper) CompleteCBPackageRefNamesFromTKR(tkr *runtanzuv1alpha3.TanzuKubernetesRelease, clusterBootstrap *runtanzuv1alpha3.ClusterBootstrap) error {
 	var suffix = "*"
+
+	if clusterBootstrap.Spec == nil {
```

Comment:
This is not strictly related to the issue this PR solves, but it did run into null pointer error when writing unit tests, so best to add this fix here.